core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -95,12 +120,14 @@ object Controller {
 
     // akka-style factory to create a Controller object
     private class ServiceBuilder(config: WhiskConfig, instance: Int, logging: Logging) extends Creator[Controller] {
-        def create = new Controller(config, instance, logging)
+        // this method is not reached unless ExecManifest was initialized successfully
+        val runtimes = ExecManifest.runtimesManifest.getOrElse(Runtimes(Set()))
```

Comment:
Should we have it throw an Exception then (or handle the `None` case appropriately) instead of defaulting to an empty set?